akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/JoinConfigCompatCheckerClusterShardingSpec.scala

```diff
@@ -29,7 +27,6 @@ object JoinConfig {
       akka.cluster.sharding.state-store-mode = "persistence"
       akka.cluster.configuration-compatibility-check.enforce-on-join = on
       akka.cluster.jmx.enabled = off
-      akka.coordinated-shutdown.terminate-actor-system = on
```

Comment:
Not even needed. The untyped version of the incompatible test uses it but it seems unnecessary.